### PR TITLE
send userReportIssue reports to Datadog

### PR DESCRIPTION
**Pull request by dhenkel92**
This PR will send the `userReportIssue` reports to Datadog.

closes https://github.com/corona-school/project-user/issues/688

### CODE REVIEW

File: graphql/user/mutations.ts [hunk 1]
```diff
@@ -84,7 +84,17 @@ export class MutateUserResolver {
         section('LOGS', 1);
         result += logs.join('\n');
 
-        logger.info(`An issue was reported from the frontend: \n` + result);
+        const logger = getLogger('IssueReporter');
+        logger.addContext('issureTag', issueTag);
+        logger.addContext('userAgent', userAgent);
```
Comment (dhenkel92) [inline graphql/user/mutations.ts:89]: @Jonasdoubleyou any idea what context might be useful from the `GraphQLContext`?

Comment (Jonasdoubleyou) [reply graphql/user/mutations.ts:89]: The userID, roles and the sessionID, the sessionID is the same we log in all the GraphQL queries.

File: graphql/user/mutations.ts [hunk 2]
```diff
@@ -27,16 +25,16 @@ class SupportMessage {
...
-    @Mutation((returns) => Boolean)
+    @Mutation(() => Boolean)
     @Authorized(Role.UNAUTHENTICATED)
     @RateLimit('Email Availability', 50 /* requests per */, 5 * 60 * 60 * 1000 /* 5 hours */)
     async isEmailAvailable(@Arg('email') email: string) {
         return await isEmailAvailable(validateEmail(email));
     }
 
-    @Mutation((returns) => String)
+    @Mutation(() => String)
```
Comment (Jonasdoubleyou) [inline graphql/user/mutations.ts:37]: using the parameter name to add some meaning to the function is in line with TypeGraphQL's documentation (https://typegraphql.com/docs/resolvers.html) and we followed that pretty consistently. So I would prefer to leave them here.

Comment (dhenkel92) [reply graphql/user/mutations.ts:37]: I'll add them back, but tbh I don't see the real benefit of having dangling variables for the sake of explanation. Furthermore, it seems to lead to forgotten arguments/variables as the linter cannot care about them right now.

**Review by Jonasdoubleyou — APPROVED** [review]

**Review by Jonasdoubleyou — APPROVED** [review]

File: graphql/user/mutations.ts (hunk 2 above)
Comment (Jonasdoubleyou) [reply graphql/user/mutations.ts]: True, also I don't see a reason why it is a function ... Anyways, we can discuss this, but then it should be done consistently across all resolvers

**Pull request merged**